data/mods/Xedra_Evolved/items/corpses.json

```diff
@@ -37,5 +37,21 @@
     "weight": "40 kg",
     "volume": "150 L",
     "flags": [ "TRADER_AVOID", "NO_REPAIR" ]
+  },
+  {
+    "type": "GENERIC",
+    "id": "broken_exodii_quad_jotunn",
+    "symbol": ",",
+    "color": "light_gray",
+    "name": { "str": "broken Exodii quadruped" },
+    "category": "other",
+    "description": "A broken Exodii Jotunn walker.  Still looks incredibly intimidating despite being permanently inoperative, due to the sheer size, mass, and protruding weapons.  Could be gutted for parts.",
```

Comment:
> Could be gutted for parts
 Should add a disassembly recipe then?

Reply:
Has a harvest like the normal exodii quad, so if you dissect it you can get cbms.   I'll check and see what all the broken_quad corpse ties into though and make sure I'm not just adding another error :)

Reply:
You were correct, it also needed a deconstruct. Thanks!